std_msgs/Header header
float32 sample_time
int32 sample_num
float32 temperature
float32 bottom_track_x
float32 bottom_track_y
float32 bottom_track_z
float32 bottom_track_q
float32 bottom_track_depth
float32 water_mass_x
float32 water_mass_y
float32 water_mass_z
float32 water_mass_q
float32 water_mass_depth
